### DISCUSSION TITLE
最新版本的bigInt类型不被node 9.11支持

### DISCUSSION THREAD

**Discussion by cnsdwu** [1715173329/UnblockNeteaseMusic#254] [Q&A] [answered]
目前安卓使用node能找到的最高版本为9.11，运行时提示bigInt is not a function
![image](https://user-images.githubusercontent.com/23513790/136720218-ccdddf78-504c-480a-a7b7-67c55ae74230.png)

**Comment by 1715173329** [comment, marked as answer]
目前所兼容的最低版本是 Node.js 12 LTS。
你有两个选择：
1. 安装最新版本的 Termux，并安装 nodejs-lts 软件包
2. 使用最新的[杜比大喇叭β](https://github.com/nining377/dolby_beta/releases)
